src/coins.h

```diff
@@ -10,13 +10,15 @@
 #include <compressor.h>
 #include <core_memusage.h>
 #include <crypto/siphash.h>
+#include <logging.h>
```

Comment:
This is only used in the cpp file, no?

Reply:
Yup - vestige from previous version where function defs were inlined to header. Thanks.